fake_service/schema/tasks.py

```diff
@@ -0,0 +1,17 @@
+from celery import shared_task
+from django.core.cache import cache
+
+from schema.services import FakeDataWrite
+
+
+@shared_task()
+def download_dataset_task(cache_task_key: str, schema_id: str, count: str, user_id: str):
+    try:
+        cache.set(cache_task_key, {"is_importing": False, "errors": []}, timeout=300)
```

Comment:
why you set `"is_importing"` to cache? i think you task doesn't "import" anything